src/parser/jobs/blm/modules/Gauge.js

```diff
@@ -154,19 +156,21 @@ export default class Gauge extends Module {
 	}
 
 	onGainPolyglot() {
-		if (this._hasPolyglot) {
-			this._overwrittenFoul++
+		this._polyglotStacks++
+		if (this._polyglotStacks > MAX_POLYGLOT_STACKS) {
+			this._overwrittenPolyglot++
 		}
-		this._hasPolyglot = true
+		this._polyglotStacks = Math.max(this._polyglotStacks, MAX_POLYGLOT_STACKS)
```

Comment:
It needs a a math.min. If you get a 3rd stack but your MAX_POLYGLOT_STACKS = 2 then it says 3 here instead of 2.

Reply:
I am a dumb. Fixed.